core/src/main/resources/hudson/PluginManager/installed.jelly

```diff
@@ -115,7 +115,15 @@ THE SOFTWARE.
                       </form>
                     </j:if>
                   </td>
-                  <td class="center pane uninstall">
+                  <j:choose>
+                      <j:when test="${p.hasMandatoryDependents() or p.hasMandatoryDependents()}">
+                            <j:set var="uninstallPossible" value="true"/>
+                      </j:when>
+                      <j:otherwise>
+                          <j:set var="uninstallPossible" value="false"/>
+                      </j:otherwise>
+                  </j:choose>
+                  <td class="center pane uninstall" data="${uninstallPossible}">
                     <j:choose>
                       <j:when test="${p.isDeleted()}">
                         <p>${%Uninstallation pending}</p>
```

Comment:
Would it make sense to distinguish between "Can be uninstalled", "cannot be uninstalled", and "has already been uninstalled"?

Reply:
Yes, this would be good, although if done it needs to be visibly distinguishable...